dev-python/dateutil/dateutil-2.8.1.recipe

```diff
@@ -45,6 +45,9 @@ REPLACES_python="
 	python_dateutil
 	"
 
+SETUPTOOLS_SCM_PRETEND_VERSION=$portVersion
+export SETUPTOOLS_SCM_PRETEND_VERSION
```

Comment:
Stumbled upon a simular problem when checking up on zipp(_python), could you check if bumping setuptools to version 42.0.0 fixes it without this fix?